src/main/java/testUI/elements/WaitFor.java

```diff
@@ -425,8 +416,7 @@ public Element untilHasNotValue(String value) {
         }
         lastCommandTime = (System.currentTimeMillis() - t);
         setTime(lastCommandTime);
-        putLogDebug("Element '" + stringElement + "' has no value '" + value + "' after "
-                + lastCommandTime + " ms");
+        putLogDebug("Element '%s' has no value '%s' after %d ms", stringElement, value, lastCommandTime);
```

Comment:
```suggestion
        putLogDebug("Element '%s' does not  have value '%s' after %d ms", stringElement, value, lastCommandTime);
```

Reply:
I think this can be `putLogInfo()` since this is an assertion as well.

Reply:
could you change the suggestions to use the putLogInfo isntead?